pkg/registry/manager.go

```diff
@@ -60,6 +60,12 @@ func Locate(a app.App, spec *app.RegistryRefSpec) (Registry, error) {
 		return githubFactory(a, spec)
 	case ProtocolFilesystem:
 		return NewFs(a, spec)
+	case ProtocolHelm:
+		client, err := newHelmRepositoryClient("https://kubernetes-charts.storage.googleapis.com", nil)
```

Comment:
Should this be configurable? Or at least `const`? Or will that be part of the `registry add` work?

Reply:
It should be, but this is dependent on `registry add`. So for now, it is hard coded.